src/goInstallTools.ts

```diff
@@ -115,9 +115,11 @@ function installTools(goVersion: SemVersion, missing?: string[]) {
 
 	outputChannel.appendLine(''); // Blank line for spacing.
 
+	let http_proxy = vscode.workspace.getConfiguration('http').get('proxy');
+	let https_proxy = http_proxy;  // default proxy for http & https
```

Comment:
@ibigbug  Sorry for the very late review.

Have you tested the case where http_proxy is not set in settings , but is set in the environment variable?
In that case will the value set in the environment variable make its way to the child process running the `go get` command?